x-pack/plugins/security_solution/public/management/cypress/tasks/alert_details_actions.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const openAlertDetailsView = (): void => {
```

Comment:
maybe rename to `openFirstAlertDetailsView()` since its only opening the first one?

or,
maybe add a input argument for maybe `rowIndex` (default it to `0`) and use it in conjunction with `getAlertsTableRows()` from `secreens/alerts.ts`?

Reply:
602dbc64a24124472991020ba6dbfda39408553a